README.md

```diff
@@ -178,5 +184,14 @@ should produce the following output:
 
 Run `python setup.py test` from within the project directory.
 
+
+## Deployments
+Deployments to PyPi are handled through [Travis-CI](https://travis-ci.org/WikiWatershed/tr-55).  Tag a commit for release and it will be automatically published.
+
+``` bash
+git tag [tag name] [commit sha]
```

Comment:
@hectcastro You'll still need to do a `git push --tags` to kick of the publishing to PyPi in that workflow?